gapic-generator-java/src/test/java/com/google/api/generator/gapic/composer/ComposerTest.java

```diff
@@ -172,6 +173,29 @@ void testComposePackageInfo_emptyGapicContext_returnsNull() {
     assertNull(Composer.composePackageInfo(GapicContext.EMPTY));
   }
 
+  @Test
+  void testComposeSelectively_serviceClientShouldOnlyContainSelectedMethods() {
```

Comment:
If we are only verifying `GapicContext`, we should test it in `ParserTest` as there are no changes in `Composer`.